cmake/Modules/FindTPLTHREADS.cmake

```diff
@@ -1,5 +1,5 @@
 INCLUDE(FindPackageHandleStandardArgs)
-INCLUDE("${CMAKE_CURRENT_SOURCE_DIR}/cmake/tpls/FindTPLPthread.cmake")
+FIND_PACKAGE(Threads)
```

Comment:
Remove the file if we don't use it any more

Reply:
AFAIK, it's used when building `Kokkos` as part of `Trilinos`.

Reply:
OK